### DISCUSSION TITLE
[Feature] Convert live pictures into static assets

### DISCUSSION THREAD

**Discussion by Funk66** [immich-app/immich#7154] [Feature Request]
### The feature

I keep inadvertently making and uploading live pictures from an iPhone to Immich when I actually only intend to keep static media. I have to remind myself to first deactivate the live feature on the iPhone's photo gallery app (if I hand't done so in the camera app already) and only then go to Immich and upload it. If there is any other way to do it, I don't know about it.
If I mistakenly upload the picture without doing that, I additionally have to prevent Immich from detecting a duplicate when I try to reupload the static version. If I do that by deleting the remote version first, I worry that I may accidentally delete my local copy as well.

TLDR: I think it would be nice to have the option to delete the video part of an image and keep it as a good old static image directly from the app.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4476
